jerry-core/ecma/operations/ecma-exceptions.h

```diff
@@ -28,6 +28,12 @@
  * @{
  */
 
+#ifndef JERRY_NDEBUG
```

Comment:
I have doubts then. Actually, I'd argue against this change. This could potentially change the way a script is executed depending only on the build mode of the engine. A debug build should make easier to debug the engine but not add extra features to it (and vice versa, release builds may/should be more optimized thus harder to debug but not lack engine functionality or perceivable features). For me, this goes against these concepts. But it would be good to hear the opinion of others.